index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tasnimul Hasan Portfolio</title>
+    <link href="styles.css" rel="stylesheet" type="text/css">
+    <style>
+        body{background: url("Image/backimg.png");
```

Comment:
- [ ] Let's make sure we add this background image to the `headline` section instead of the body because if it's the body it will extend as we extend our body content. 
The issue already occur if you scroll a little bit down